api/src/main/java/io/smallrye/reactive/messaging/EmitterConfiguration.java

```diff
@@ -0,0 +1,20 @@
+package io.smallrye.reactive.messaging;
+
+import org.eclipse.microprofile.reactive.messaging.OnOverflow;
+
+import io.smallrye.reactive.messaging.annotations.EmitterFactoryFor;
+
+public interface EmitterConfiguration {
```

Comment:
Didn't we have a class named EmitterConfiguration already?

Reply:
Yes, I extracted an interface and moved to the api module. It is passed to the `EmitterFactory` interface, which is also created in the api module. The provider contains the default impl `DefaultEmitterConfiguration`, similarly to `MediatorConfiguration`.